### DISCUSSION TITLE
Message and image come in 2 different messages

### DISCUSSION THREAD

**Discussion by daniltreshchev** [grafana/grafana#55038] [Alerting]
Message and image come in 2 different messages

I’m using a remote render server, how can I make the alert and picture be in 1 post instead of 2 different ones

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
